test/integration/node-specific/client_close.test.ts

```diff
@@ -205,10 +542,10 @@ describe.skip('MongoClient.close() Integration', () => {
             await runScriptAndGetProcessInfo(
               'tls-file-read-auto-encryption',
               config,
-              async function run({ MongoClient, uri, expect, ClientEncryption, BSON }) {
+              async function run({ MongoClient, uri, expect, mongodb }) {
                 const infiniteFile = '/dev/zero';
 
-                const kmsProviders = BSON.EJSON.parse(process.env.CSFLE_KMS_PROVIDERS);
+                const kmsProviders = mongodb.BSON.EJSON.parse(process.env.CSFLE_KMS_PROVIDERS);
```

Comment:
This test fails locally when CSFLE_KMS_PROVIDERS isn't defined, which is the default, can you put the metadata on this test to have it skip unless encryption is enabled?